packages/manager/apps/dedicated/client/app/license/license.html

```diff
@@ -126,7 +126,7 @@
                 </oui-action-menu-item>
                 <oui-action-menu-item
                     data-ng-if="$row.type !== licenseTypes.SPLA && !$row.isExpired"
-                    on-click="setAction('delete/license-delete', { 'license': $row })"
+                    on-click="setAction('delete/license-delete', { 'license': $row }); trackDelete()"
```

Comment:
Maybe a function should be created to avoid chaining function calls in `on-click`